src/smolvm/telemetry/__init__.py

```
@@ -0,0 +1,26 @@
+"""VM-level telemetry collection and storage."""
+
+from smolvm.telemetry._types import BootEvent, BootStatus, CommandExecution, VMTelemetry
+from smolvm.telemetry.manager import TelemetryManager
+
+__all__ = [
+    "BootEvent",
+    "BootStatus",
+    "CommandExecution",
+    "VMTelemetry",
+    "TelemetryManager",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)